apps/arc/little-arc_mac.talon

```diff
@@ -0,0 +1,7 @@
+user.running: arc
+-
```

Comment:
These need to match `os: mac` to be Mac-specific.

Reply:
Thanks - changed.